vspace/src/main/webapp/WEB-INF/views/staff/modules/slides/slide.html

```diff
@@ -361,6 +406,9 @@
 					contentType: false,
 					processData: false,
 					data: formData,
+					headers:{
+            			'X-CSRF-Token':token
+            		},
```

Comment:
?

Reply:
you said that the CSFR token should not be directly used in the URL but should be passed through header, thats why I passed the token through header.

Reply:
this is about indentation